Docs: removes-yum (#70634)
grafana/grafana: docs/sources/setup-grafana/installation/redhat-rhel-fedora/index.md

This topic explains how to install Grafana dependencies, install Grafana on RHEL or Fedora, and start the Grafana server on your system.

> You can install Grafana using RPM or by downloading a binary `.tar.gz` file.

Reviewer: ```suggestion
You can install Grafana from the RPM repository, from standalone RPM, or with the binary `.tar.gz` file.
```

If you install via RPM or the `.tar.gz` file, then you must manually update Grafana for each new version.

## Install Grafana from the RPM repository

> If you install from the RPM repository, then Grafana is automatically updated every time you run `sudo rpm update`.

Reviewer: ```suggestion
If you install from the RPM repository, then Grafana is automatically updated every time you update your applications.
```

Grafana Enterprise is the recommended and default edition. It is available for free and includes all the features of the OSS edition. You can also upgrade to the [full Enterprise feature set](/products/enterprise/?utm_source=grafana-install-page), which has support for [Enterprise plugins](/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
{{% /admonition %}}

> To install Grafana using a RPM repository, complete the following steps:

Reviewer: ```suggestion
To install Grafana from the RPM repository, complete the following steps:
```

   sudo rpm --import gpg.key
   ```

1. Add a file to your RPM repository using the method of your choice.

   The following example uses `nano` to add a file to the RPM repo.

   ```bash

>    sudo nano /etc/yum.repos.d/grafana.repo

Reviewer: I think we could clean this up and simplify a little and replace it all with:

> Create `/etc/yum.repos.d/grafana.repo` with the following content:
